android/src/main/java/com/mapbox/reactnativemapboxgl/ReactNativeMapboxGLView.java

```diff
@@ -88,15 +89,19 @@ public ReactNativeMapboxGLView(Context context, ReactNativeMapboxGLManager manag
     // Lifecycle methods
 
     public void onAfterUpdateTransaction() {
-        if (_mapView != null) { return; }
+        if (_mapView != null) {
```

Comment:
Why did you unnecessarily change formatting of the code in this file? These `if` statements just return. There's no need to put them on 3 lines instead of just one.